scripts/verify-bun-review-artifacts.py

```
@@ -8064,6 +8092,17 @@ def sanitize_artifact_name(value: str) -> str:
     return f"{sanitized[:prefix_len]}-{digest[:ARTIFACT_NAME_HASH_CHARS]}"
 
 
+def sanitize_lane_artifact_name(value: str) -> str:
+    if not value.strip():
+        fail("lane artifact identity must not be empty")
+    sanitized = sanitize_artifact_name(value)
+    if all(ch.isascii() and ch.isalnum() or ch in "-_" for ch in value):
```

Ruff (0.16.1)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)